static/src/javascripts/projects/common/utils/detect.js

```diff
@@ -405,38 +409,58 @@ define([
     // this is soon - it's not worth refactoring them when they're off soon
     //
     // ** don't forget to remove them from the return object too **
-    function adblockInUseSync() {
-        if (!detect.cachedAdblockInUse) {
-            var sacrificialAd = createSacrificialAd(),
-                contentBlocked = isHidden(sacrificialAd);
-            sacrificialAd.remove();
-            detect.cachedAdblockInUse = contentBlocked;
-            return contentBlocked;
-        }
+    var adblockInUseSync, getFirefoxAdblockPlusInstalledSync, createSacrificialAd;
+    if (userPrefs.get('lighter-adblock')) {
```

Comment:
I actually probably wouldn't bother putting this in a separate branch - I'd just replace the original function entirely. This also make it easy to check the new approach remains accurate in the wild - we just have to look at Ophan and check adblockers are still being detected.